### DISCUSSION TITLE
Multicast MPEGTS

### DISCUSSION THREAD

**Discussion by elliotd123** [OvenMediaLabs/OvenMediaEngine#1211] [Help]
Is there a way to join to multicast streams by IP address? It seems like the configuration only supports binding to a port. This may be ok in some situations, but what about enviroments where there are multiple multicast streams with the same port?

**Comment by elliotd123** [comment]
To add to this - It doesn't look like OME binds to multicast - Unless there's some hidden configurations for mpegts - Is this possible at all?

**Comment by getroot** [comment]
Unfortunately, OvenMediaEngine does not yet support media sources from multicast. It is recommended to use an external tool such as ffmpeg to pull the multicast source and then input rtmp, mpeg2-ts, or srt to OME.
